src/include/duckdb/parser/parsed_data/exported_table_data.hpp

```diff
@@ -0,0 +1,35 @@
+//===----------------------------------------------------------------------===//
+//                         DuckDB
+//
+// duckdb/parser/parsed_data/export_table_data.hpp
+//
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "duckdb/parser/parsed_data/parse_info.hpp"
+#include "duckdb/common/types/value.hpp"
+
+namespace duckdb {
+
+struct ExportedTableData : public ParseInfo {
```

Comment:
As stated above, I recommend having just a single node (`BoundExportData`) that holds the bind data for all the tables. The BoundExportTable can then inherit from ParseInfo, but the data for each of the individual tables does not need to (i.e. this can just be a regular struct with two string members).